community-plugins.json

```diff
@@ -5788,5 +5788,13 @@
         "author": "Ryan Gomba",
         "repo": "ryangomba/obsidian-todo-sort",
         "branch": "main"
+    },
+    {
+        "id": "obsidian-console",
```

Comment:
While there are a lot of plugins that include "obsidian" in the ID, it's not necessary to include it.

Reply:
I'm much too late with my response. Yes, if possible, change the ID to a shorter one. Thank you!

Reply:
You'll need to change the name/ID on your end since they need to be changed here and in your plugin repo to match.